### PR TITLE
CLOUDP-238529: Support export streams resource in generate and apply commands

### PR DESCRIPTION
**Pull request by helderjs**
## Proposed changes

This PR adds support to export streams resources recently added in AKO v2.3.0 in both `generate` and `apply` commands

_Jira ticket:_ CLOUDP-238529

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — APPROVED** [review]
LGTM if LGTEVG but double checking k8s tests break every-time there's a new release until this PR is merged, is there any way to prevent that ?

**Comment by helderjs** [comment]
> LGTM if LGTEVG but double checking k8s tests break every-time there's a new release until this PR is merged, is there any way to prevent that ?

@gssbzn Thanks! I have added a ticket for us to track and solve this issue

File: internal/kubernetes/operator/features/crds.go
Comment (josvazg) [inline internal/kubernetes/operator/features/crds.go:57]: @s-urbaniak the nop patcher is so prominent that we might want to make it the default. So if nothing is set, that one is the one used. Will avoid a lot of changes that are not needed. We would only update when patching is required.

**Review by josvazg — APPROVED** [review]

**Pull request merged**
